server/handlers/products.js

```diff
@@ -0,0 +1,33 @@
+var dao = require("../dao/dao");
+const { Product } = require("../models/product");
+const {
+    searchStringValidator,
+    productCategoryValidator,
+    IDsValidator
+} = require("./shared_validators");
+const { validationResult } = require("express-validator");
+
+exports.getProductsByIDValidatorChain = [searchStringValidator, productCategoryValidator, IDsValidator];
+
+exports.getProductsByIDHandler = async function (req, res, next) {
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+        return res.status(400).json({ errors: errors.array() });
+    }
+
+    await dao
+        .getProducts(req.body.category
+            , req.body.searchString
+            , req.body.IDs)
```

Comment:
use lowercase for attributes, I don't know why it is uppercase in the api documentation
```suggestion
            , req.body.ids)
```